creusot-contracts-proc/src/lib.rs

```diff
@@ -392,14 +392,15 @@ impl Parse for Assertion {
 pub fn proof_assert(assertion: TS1) -> TS1 {
     let assert = parse_macro_input!(assertion as Assertion);
     let assert_body = pretyping::encode_block(&assert.0).unwrap_or_else(|e| e.into_tokens());
+    let location = format!("proof_assert L{}", Span::call_site().start().line);
```

Comment:
I'm trying to store the line number of each `proof_assert` here but I keep getting 0.

Reply:
Postponing to other PR